docs/calling-the-asset-delivery-api.md

```diff
@@ -0,0 +1,168 @@
+# Calling the Asset Delivery APIs
```

Comment:
Comment: I feel that there's quite a lot of repetition here, but then again, there are some differences also, so maybe the repetition is the clearest way.